vulnz/src/docker/scripts/mirror.sh

```diff
@@ -5,10 +5,28 @@ echo "Updating..."
 DELAY_ARG=""
 if [ -z $NVD_API_KEY ]; then
   DELAY_ARG="--delay=10000"
+else
+  echo "Using NVD API KEY: $NVD_API_KEY"
```

Comment:
We can't echo the full API Key. Otherwise, we will see a "security" professional try to open a CVE against the project. I get it, but to try and argue the severity just isn't worth it (me: "this is not critical, it is an API key that is free to obtain and is used to access public data", consultant: "an attacker could DoS you by using the key to reach the rate limit, critical", me: ).

```suggestion
  echo "Using NVD API KEY: ${NVD_API_KEY:0:5}****"
```

Reply:
Sure, not a big issue for me. This API key is not really sensible but well, CVE not needed :)

Reply:
Just outputting the first few characters will let you know if you are using the correct key.